server_test.go

```diff
@@ -164,6 +168,20 @@ func UDPFunc(t *testing.T) {
 	}
 }
 
+func TestAT(t *testing.T) {
```

Comment:
This test that the server accepts a message, but not that it writes a log, please test this like it's done for the NAT messages.

Reply:
I guess this function doesn't really test anything. But the log writing is confirmed in TestOutput

Reply:
Oh, that test is hard to catch.